srcalc/src/main/java/gov/va/med/srcalc/domain/model/RiskModel.java

```diff
@@ -187,9 +188,33 @@ void setNumericalTerms(Set<NumericalTerm> numericalTerms)
      * For reflection-based construction only. Business code should modify the
      * Set returned by {@link #getProcedureTerms()}.
      */
-    void setProcedureTerms(Set<ProcedureTerm> derivedTerms)
+    void setProcedureTerms(Set<ProcedureTerm> procedureTerms)
+    {
+        fProcedureTerms = procedureTerms;
+    }
+    
+    /**
+     * <p>The {@link DerivedTerm}s in the model's sum. Mutable.</p>
+     * 
+     * <p>See {@link #getTerms()} for a read-only view of all of the terms.</p>
+     */
+    @ElementCollection(fetch = FetchType.EAGER) // eager-load due to close association
```

Comment:
I think we should test derived terms in RiskModelTest somewhere. Maybe you could do this as part of the procedure complexity terms.